content/applications/general/payment_acquirers.rst

```diff
@@ -1,12 +1,238 @@
-=================
-Payment Acquirers
-=================
+:show-content:
+
+================================================
+Payment Acquirers (Credit Cards, Online Payment)
+================================================
 
 .. toctree::
    :titlesonly:
 
-   payment_acquirers/payment_acquirers
    payment_acquirers/wire_transfer
    payment_acquirers/authorize
    payment_acquirers/paypal
    payment_acquirers/stripe
+
+Odoo embeds several **payment methods** that allow your customers to pay on their *Customer Portals*
+or your *eCommerce website*. They can pay Sales Orders, invoices, or subscriptions with recurring
```

Comment:
@AntoineVDV 
why so?
Isn't it a Sales Orders that e-commerce creates, that the web visitors pay?
If it's just a quotation, then they don't have to pay.

Reply:
It's complicated because, in Odoo, sales quotations and orders are two different stages of the same model that they share. From the portal, you can only pay a sales order if it's in the stage 'quotation'. Once it reaches the stage 'order', it's confirmed and there is nothing to pay.

Reply:
But still, from a business pov... they are provided with a quotation to decided if yes or no they want to take that offer.
If they agree, it becomes an order, a contract between the customer and the provider.
So if the agreement is marked by the payment, it's understandable that what they see before is a quotation and that it becomes an order at the moment of the payment. Still, they're paying an order, and not a quotation.
Or I may be awfully wrong, who knows ^^

Reply:
That's a way to see it, yes. Well, let's not lose any more time on nitpicking details like this if both formulations are correct anyway.